Libraries/Components/DatePickerAndroid/DatePickerAndroid.android.js

```diff
@@ -52,14 +52,15 @@ class DatePickerAndroid {
    *   * `date` (`Date` object or timestamp in milliseconds) - date to show by default
    *   * `minDate` (`Date` or timestamp in milliseconds) - minimum date that can be selected
    *   * `maxDate` (`Date` object or timestamp in milliseconds) - minimum date that can be selected
+   *   * `calendarMode` (`Boolean` true or false) - To set the date-picker mode to spinner/calendar.
```

Comment:
As per the android API documentation, Android date-picker supports only two modes. https://developer.android.com/reference/android/R.styleable.html#DatePicker_datePickerMode